src/pages/Checkout.page.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-extra-semi */
+/* eslint-disable linebreak-style */
+/* eslint-disable import/prefer-default-export */
+
+const { BaseSwagLabPage } = require('./BaseSwagLab.page');
+
+export class CheckoutPage extends BaseSwagLabPage {
+    url = '/checkout-step-one.html';
+
+    get firstName() { return this.page.locator('#first-name'); }
+
+    get lastName() { return this.page.locator('#last-name'); }
+
+    get zipCode() { return this.page.locator('#postal-code'); }
+
+    get continueBtn() { return this.page.locator('.submit-button'); }
+
+    // eslint-disable-next-line linebreak-style
+    async fillCustomerData(firstName, lastName, zipCode) {
```

Comment:
```suggestion
    async fillAndSendCustomerData(firstName, lastName, zipCode) {
```
Otherwise sending data is a side effect :)

Reply:
This could be fixed like "later" as one day you may want to fill data and dont send it (recheck filling is correct), so such helper functions should be as separated as possible. But it could be JFYI, I dont think it is the blocking issue.